src/Rules/Comparison/StrictComparisonOfDifferentTypesRule.php

```diff
@@ -62,8 +62,8 @@ public function processNode(Node $node, Scope $scope): array
 				$addTip(RuleErrorBuilder::message(sprintf(
 					'Strict comparison using %s between %s and %s will always evaluate to false.',
 					$node->getOperatorSigil(),
-					$leftType->describe(VerbosityLevel::value()),
-					$rightType->describe(VerbosityLevel::value()),
+					$leftType->describe(VerbosityLevel::getRecommendedLevelByType($leftType, $rightType)),
```

Comment:
I feel like this is a breaking change. The difference: When you have `1` and `'foo'`, before it used these exact scalar values. Now it's just going to say `int` and `string`.

Reply:
Just revert the change in this rule and we'll see what fails.